### PR TITLE
fix(my collection): artwork list empty state alignment #trivial

### PR DESCRIPTION
**Pull request by admbtlr**
The type of this PR is: Bugfix

This PR resolves [CX-767]

### Description

Moves the zero state component to the centre of the screen. Also reduces the font size.

image

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-767]: https://artsyproduct.atlassian.net/browse/CX-767

### CODE REVIEW

File: src/lib/Components/States/ZeroState.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const ZeroState = (props: ZeroStateProps) => (
 
     {!!props.subtitle && (
       <>
-        <Sans size="4" textAlign="center">
+        <Sans size="4" textAlign="center" maxWidth={!!props.title ? "100%" : "80%"}>
```
Comment (ds300) [inline src/lib/Components/States/ZeroState.tsx:21]: nitpick, nonblocking:
```suggestion
        <Sans size="4" textAlign="center" maxWidth={props.title ? "100%" : "80%"}>
```

File: src/lib/Components/States/ZeroState.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from "react"
 import { Flex, Sans, Spacer } from "palette"
 
 interface ZeroStateProps {
-  title: string
+  title?: string
```
Comment (ds300) [inline src/lib/Components/States/ZeroState.tsx:6]: seems like if the title is not given we should hide the associated Sans and the Spacer below. Curious how it'd affect this layout issue. Did you explore that?

Comment (admbtlr) [reply src/lib/Components/States/ZeroState.tsx:6]: Good point - I've changed it, and changed the padding to reflect the removed spacer

**Comment by ds300** [comment]
oops sorry I hit squash and merge, still got that muscle memory 😓

**Comment by admbtlr** [comment]
No no, that was right!

**Comment by ds300** [comment]
Hmm but you assigned yourself, so it should be your responsibility to hit the green button. No?

**Comment by admbtlr** [comment]
Oh, I see what you mean - yeah true, but actually I only assigned myself out of muscle memory, so it's all good. I should probably get used to assigning a reviewer instead.

**Pull request merged**